docs/glsl-fuzz-walkthrough.md

```diff
@@ -47,6 +47,43 @@ Add the following directories to your path:
 The `graphicsfuzz/` directory is the unzipped release.
 If building from source, this directory can be found at `graphicsfuzz/target/graphicsfuzz/`.
 
+### amber
+
+In March 2019 we depreciated our legacy vulkan worker to instead rely on
+[amber](https://github.com/google/amber) by default.
+
+For testing Vulkan drivers on Android, you need to build amber as a native
+Android executable, and place that executable in `/data/local/tmp` on any device
+where you want to run tests.
+
+As of March 2019, building amber as a native executable should require the
+following (amber commit `f0a3613b727c240ae8b98e232e655db71d87217e` is known to
+work):
+
+```
+git clone git@github.com:google/amber
+cd amber
+./tools/git-sync-deps
+./tools/update_build_version.py . samples/ third_party/
+mkdir build
+cd build
+mkdir app
```

Comment:
Not a big deal, but I found that making these directories is not necessary, so we could remove these steps.